src/snc/agents/hedgehog/strategic_idling/strategic_idling_hedgehog_gto.py

```diff
@@ -84,6 +84,7 @@ def get_allowed_idling_directions(self, state: StateSpace) -> StrategicIdlingOut
         :return: set of allowed idling resources with auxiliary variables
         """
         w = self._workload_mat @ state
+        self._safety_stocks_vec = safety_stocks_vec
```

Comment:
This seems out of nowhere here and difficult to know when it will be used. Can we do anything about that?